Makefile

```diff
@@ -1,2 +1,110 @@
+IMG ?= quay.io/che-incubator/che-workspace-controller:7.1.0
+NAMESPACE ?= che-workspace-controller
+TOOL ?= oc
+CLUSTER_IP ?= 192.168.99.100
+PULL_POLICY ?= Always
+WEBHOOK_ENABLED ?= false
+
+all: help
+
+_set_context:
+	$(TOOL) create namespace $(NAMESPACE) || true
+ifeq ($(TOOL),oc)
+	$(TOOL) project $(NAMESPACE)
```

Comment:
Yeah, I agree that this could use some work. The reason I did it in this way is to make sure we're in a consistent state after the rule is finished:
- If no project exists, we have to create it
- If the project exists, then we don't switch to it automatically

This means we know we're in the defined namespace if the project didn't exist, but if the project already exists, our current project could be anything. I didn't really consider that the namespace is hard-coded throughout though (and only tested the code with the default namespace)

Reply:
I've taken out bits about choosing a namespace but left it in the makefile for now -- WDYT?

Reply:
Also: removed setting kube context and OpenShift project.

Reply:
:+1: